Don't use the xlocale.h header since it is an OS package and not always provided.
Author: R. van Dijk, 2018-03-07
Updated by: Orient Kružík (INUITS)
--- migrate-5.0.3_orig/src/src/data.c	2022-04-03 00:11:20.000000000 +0200
+++ migrate-5.0.3/src/src/data.c	2022-05-09 17:17:46.836503576 +0200
@@ -53,9 +53,6 @@
 
 #include <string.h>
 #include <wchar.h>
-#ifndef WIN32
-#include <locale.h>
-#endif
 #include "migration.h"
 #include "sighandler.h"
 #include "tools.h"